Cargo.toml

```
@@ -16,3 +16,7 @@ std = []
 
 [dependencies]
 cfg-if = "0.1"
+serde = { version = "1.0.89", optional = true }
```

nit: I don't like to have the patch version in dependencies.

```suggestion
serde = { version = "1.0", optional = true }
```